tests/testthat/helper.R

```diff
@@ -102,3 +102,13 @@ skip_on_R42_win <- function() {
     testthat::skip("Long path is not supported by this version of Rtools.")
   }
 }
+
+skip_if_opted_out_of_dev_tests <- function() {
+  env_var <- Sys.getenv("REXTENDR_SKIP_DEV_TESTS") |>
+    stringi::stri_trim_both() |>
+    stringi::stri_trans_tolower()
```

Comment:
Maybe we want to do this everywhere we import environment variables. Maybe it is too cautious here. Either way.

Reply:
I checked other cases and we do not want the same thing everywhere since there are some sensitive configs. Trimming will not harm but lowercasing might be a stretch. I would say it is out of scope of this pr